src/main/scala/where/Condition.scala

```diff
@@ -76,3 +77,12 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class NoRelationshipCondition(node: Node) extends Condition {
+  def toQuery(identifiableMap: IdentifiableMap): String = {
+    val identifier = identifiableMap.get(node) getOrElse (throw new IdentifierDoesntExistException())
+    "NOT (" + identifier + ")-[]-()"
```

Comment:
use interpolation since it reads nicer in this case: `s"NOT ($identifier)-[]-()"`